### PR TITLE
New Pages and static site generator

### PR DESCRIPTION
**Pull request by ThePeach**
# tl;dr:

The current set of changes should be available at http://courier-lizard-31583.netlify.com/ (maybe)

The updated `/dist` directory now contains 

- updated index with latest changes introduced in previous PR (making it obsolete now)
- two new pages:
  - **speakers** with link to call for talk
  - **location** with link to google maps and embedded map

## Changelog

- replaced node module `rimraf` (now deprecated) with `del`
- added dependencies for metalsmith
- added gulp task for metalsmith
- split `app.scss` into different partials
- updated copy on index page
- added `location` page
- added `speakers` page
- updated `/dist` content

### CODE REVIEW

**Comment by oscarduignan** [comment]
Here's a screenshot of where the deploy preview can be viewed, number 1 toggles the details and number 2 opens the preview

image
